src/components/Badge/Badge.scss

```
@@ -18,7 +18,7 @@ $pip-spacing: ($height - $pip-size) / 2;
   display: inline-flex;
   align-items: center;
   padding: 0 $horizontal-padding;
-  background-color: var(--p-interactive-neutral-elevation-1, color('sky'));
+  background-color: var(--p-neutral-action-disabled, color('sky'));
```

feels like the wrong role to use here

---

I feel the same way, but I'm not sure what actually fits there.

---

Probably a good use-case for the base neutral

```suggestion
  background-color: var(--p-neutral, color('sky'));
```